lib/pharos/phases/configure_kubelet.rb

```diff
@@ -93,6 +93,15 @@ def existing_dropin
       def build_systemd_dropin
         options = []
         options << "Environment='KUBELET_EXTRA_ARGS=#{kubelet_extra_args.join(' ')}'"
+
```

Comment:
I think we should have specs for `build_systemd_dropin` (because it starts to be a pretty complex).

Reply:
Added